sdk/storage/Azure.ResourceManager.Storage/src/Customized/Models/BlobRestoreContent.Serialization.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Storage.Models
```

Comment:
please open another issue to track this customization which will be removed once the `2022-05-01` is merged

Reply:
I use directive instead of customization to solve this problem and this directive has no side impact if this issue is fixed in swagger side.